Makefile

```
@@ -374,6 +374,7 @@ AFLAGS_KERNEL	=
 LDFLAGS_vmlinux =
 CFLAGS_GCOV	:= -fprofile-arcs -ftest-coverage -fno-tree-loop-im $(call cc-disable-warning,maybe-uninitialized,)
 CFLAGS_KCOV	:= $(call cc-option,-fsanitize-coverage=trace-pc,)
+CFLAGS_KCOV	+= $(call cc-option,-fsanitize-coverage=trace-cmp,)
```

I think we will need a separate config for this. Some users do not need comparisons, but the instrumentation slows down execution.
If config is not enabled the corresponding ioctl should return an error.